base/.eslintrc.json

```diff
@@ -26,7 +26,7 @@
     "no-self-compare": 2,
     "no-useless-call": 2,
     "no-shadow-restricted-names": 2,
-    "no-shadow": 2,
+    "no-shadow": 0,
```

Comment:
This also needs moving into the typescript overrides.

Reply:
Done